### PR TITLE
Description added for patron workflow nodes

### PR DESCRIPTION
**Pull request by rmathew1011**

### CODE REVIEW

File: patron/nodes/others.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "id": "eda4f0e5-b9c0-4e2d-985b-9894e8179425",
   "name": "Query Other Patrons",
-  "description": "",
+  "description": "Executes queries on other patrons in divIt database.",
```
Comment (kaladay) [inline patron/nodes/others.json:4]:
```suggestion
  "description": "Executes queries on other patrons in DivIt database.",
```

File: patron/nodes/students.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "id": "18d0e1eb-46c2-4a9e-8838-88584d7998cd",
   "name": "Query Student Patrons",
-  "description": "",
+  "description": "Executes query on 'student' patrons from 'divit' database.",
```
Comment (kaladay) [inline patron/nodes/students.json:4]:
```suggestion
  "description": "Executes query on student patrons from DivIT database.",
```

File: patron/nodes/studentsConnectTo.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "id": "50a69e06-40e2-4d8e-8f1d-e6db9344cfb0",
   "name": "Complete Students",
-  "description": "",
+  "description": "Connects to parallel gateway to complete 'student' task.",
```
Comment (kaladay) [inline patron/nodes/studentsConnectTo.json:4]:
```suggestion
  "description": "Connects to parallel gateway to complete student task.",
```

File: patron/nodes/studentsMoveToNode.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "id": "76bff6c3-20be-4645-b7c6-8865ccd25603",
   "name": "Students",
-  "description": "",
+  "description": "Routes workflow to 'Students' node, connects to gateway, executes query and connection task.",
```
Comment (kaladay) [inline patron/nodes/studentsMoveToNode.json:4]:
```suggestion
  "description": "Routes workflow to students node, connects to gateway, executes query and connection task.",
```

File: patron/nodes/transform.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   "id": "96286fc0-6663-11eb-b98e-f1ccdde4a279",
   "name": "Transform Results",
-  "description": "",
+  "description": "Exceutes Javascript to process patron objects.",
```
Comment (kaladay) [inline patron/nodes/transform.json:4]:
```suggestion
  "description": "Transforms the database results into processable objects.",
```

**Review by kaladay — CHANGES_REQUESTED** [review]

File: patron/nodes/others.json (hunk 1 above)
Comment (rmathew1011) [reply patron/nodes/others.json:4]: Changes made.

File: patron/nodes/students.json (hunk 2 above)
Comment (rmathew1011) [reply patron/nodes/students.json:4]: Changes made.

File: patron/nodes/studentsConnectTo.json (hunk 3 above)
Comment (rmathew1011) [reply patron/nodes/studentsConnectTo.json:4]: Changes made.

File: patron/nodes/studentsMoveToNode.json (hunk 4 above)
Comment (rmathew1011) [reply patron/nodes/studentsMoveToNode.json:4]: Changes made.

File: patron/nodes/transform.json (hunk 5 above)
Comment (rmathew1011) [reply patron/nodes/transform.json:4]: Changes made.

**Review by kaladay — APPROVED** [review]

**Pull request merged**